device/transport/mqtt/test/_mqtt_test.js

```diff
@@ -295,6 +295,21 @@ describe('Mqtt', function () {
         fakemqtt.emit('connect', { connack: true });
       });
 
+      /*Tests_SRS_NODE_DEVICE_MQTT_41_004 [ The `sendEvent` method shall call its callback with a `MessageEnqueued` ]*/
+      /*Tests_SRS_NODE_DEVICE_MQTT_41_005 [ The `sendOutputEvent` method shall call its callback with a `MessageEnqueued` ]*/
+      it('calls the callback with the correctly structured response parameter', function (done) {
+        var transport = new Mqtt(fakeAuthenticationProvider, fakeMqttBase);
+        let foo = new results.MessageEnqueued();
```

Comment:
>let foo = new results.MessageEnqueued(); [](start = 8, length = 40)

This confuses me.  It doesn't appear to be used anywhere.  Further, don't use foo as a var name.

Reply:
good catch. left that in there.